src/pytypes/onefuzztypes/models.py

```diff
@@ -419,11 +419,16 @@ class Pool(BaseModel):
     pool_id: UUID = Field(default_factory=uuid4)
     os: OS
     managed: bool
+    max_size: int
```

Comment:
In addition to max size, can you add min_size?  While we don't use it yet, we want to make use of pools of VMs for repro  & regression testing, and having an option for those to always available would be beneficial for user interaction.